llpc/patch/llpcPatchBufferOp.cpp

```diff
@@ -1521,31 +1520,58 @@ bool PatchBufferOp::RemoveUsersForInvariantStarts(
 }
 
 // =====================================================================================================================
-// Replace a fat pointer load with the intrinsics required to do the load.
-Value* PatchBufferOp::ReplaceLoad(
-    LoadInst* const pLoadInst) // [in] The load instruction to replace.
+// Replace a fat pointer load or store with the required intrinsics.
+Value* PatchBufferOp::ReplaceLoadStore(
```

Comment:
Can the return value not be an `Instruction`? If not, consider changing this to return `Instruction`.

Reply:
Seems like `BitCast` and `InsertElement` only generate `Value`s, not `Instruction`s, so this has to return a `Value`.